### ISSUE TITLE
Package Documentation Page ?

### ISSUE THREAD

**Issue by SharathGa** [uptake/numpynet#54]
It would be useful if we add a documentation page for the library. 

Any specific reason/decisions for just having the README?

@Chronocook @jameslamb @bburns632

**Comment by bburns632** [comment]
Hi @SharathGa , see notes in #50.  

In short:
1. A PR that enforces a style must make changes to adhere current code to that style in order to not have master fail every build. 
2. Because of that, I recommend we start with code style polices that impact the current code minimally.

**Comment by Chronocook** [comment]
Agreed, no reason to not have proper docs.  They'd should be pretty light.  My eventual goal would be to have a few nice notebooks alongside the code so that someone who is learning could step through and learn one major concept per notebook.  The first notebook could just be the basic feed forward and backprop concepts. Then we could get into other concepts like learning rate, etc..

**Comment by SharathGa** [comment]
@Chronocook 

Got it, I will come up with a light weight docs for the package and you could take a look and suggest changes according to the scope of the package. I also like the idea of having a jupyter notebooks for individual concepts.
